frontend/packages/topology/src/actions/contextMenuActions.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Node } from '@patternfly/react-topology';
+import { ActionsLoader } from '@console/shared';
+import { createContextMenuItems } from '../components/graph-view';
+
+export const contextMenuActions = (element: Node): React.ReactElement[] => {
+  return [
+    <ActionsLoader
+      key={`topology-actions-${element.getType()}`}
+      contextId="topology-actions"
```

Comment:
We also need `resource` `contextId` for this util to be reused for other nodes.

@rohitkrai03 we need a way to avoid doing it this way and have a single loader

Reply:
Yeah, I am working on adding support for resource actions in action loader. So we can provide the resource model along with contextId to the action loader and get back all the actions for a specific resource as well as a context.

Reply:
do we put this PR on hold till then ?

Reply:
I think we can get this merged now. The follow up story that I am working on will take care of resource actions.